src/library-authoring/library-sidebar/index.js

```diff
@@ -0,0 +1,3 @@
+// @ts-check
```

Comment:
Need to migrate to `tsx` here
```suggestion
```

Reply:
Updated https://github.com/open-craft/frontend-app-course-authoring/pull/38/commits/d37ffbf4000b14cfe5ea3065056e7c3d1da1f4e5